content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1109 @@
+---
+title: RBAC認可を使用する
+content_type: concept
+aliases: [/rbac/]
+weight: 70
+---
+
+<!-- overview -->
+Role Based Access Control(RBAC)は、組織内の個々のユーザーのRoleをベースに、コンピューターまたはネットワークリソースへのアクセスを制御する方法です。
+
+
+<!-- body -->
+RBAC認可は{{< glossary_tooltip text="API group" term_id="api-group" >}} `rbac.authorization.k8s.io`を使用して認可の決定を行い、Kubernetes APIを介して動的にポリシーを構成できるようにします。
+
+RBACを有効にするには、以下の例のように{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}の`--authorization-mode` フラグをコンマ区切りの`RBAC`を含むリストでスタートします。
+
+```shell
+kube-apiserver --authorization-mode=Example,RBAC --other-options --more-options
+```
+
+## APIオブジェクト{#api-overview}
+
+RBAC APIは4種類のKubernetesオブジェクト(_Role_、 _ClusterRole_、  _RoleBinding_ そして _ClusterRoleBinding_)を宣言します。他のKubernetesオブジェクトのように`kubectl`のようなツールを使って、[オブジェクトを記述](/docs/concepts/overview/working-with-objects/kubernetes-objects/#understanding-kubernetes-objects)、または変更できます。
+
+{{< caution >}}
+これらのオブジェクトは設計上、アクセス制限を課します。学んできたように変更を行っている場合、
+[特権エススカレーション防止とブートストラップ](#特権昇格の防止とブートストラップ)
+を参照し、これらの制限がどのようにいつかの変更を防止するのかを理解しましょう。
+{{< /caution >}}
+
+### RoleとClusterRole
+
+RBACの _Role_ または _ClusterRole_ には、一連の権限を表すルールが含まれて言います。
+権限は完全な追加方式です。("deny"のルールはありません)。
+
+Roleは常に特定の{{< glossary_tooltip text="namespace" term_id="namespace" >}}で権限を設定します。
+つまり、Roleを作成する時は、Roleが属するNamespaceを指定する必要があります。
+
+対照的にClusterRoleは、Namespaceに属さないリソースです。Kubernetesオブジェクトは常にNamespaceに属するか、属さないのいずれかである必要があり、リソースは異なる名前(RoleとClusterRole)を持っています。つまり、両方であることは不可能です。
+
+ClusterRolesにはいくつかの用途があります。ClusterRoleを利用して、以下のことができます。
+
+1. Namespaceに属するリソースに対する権限を定義し、個々のNamespace内で付与する
+2. Namespaceに属するリソースに対する権限を定義し、すべてのNamespaceにわたって付与する
+3. クラスター単位でスコープされているリソースに対するアクセス許可を定義する
+
+NamespaceでRoleを定義する場合は、Roleを使用します。クラスター全体でRoleを定義する婆は、ClusterRoleを使用します
+
+#### Roleの例
+
+以下はNamespace「default」にあるRoleの例で、
+{{< glossary_tooltip text="pods" term_id="pod" >}}への読み取りアクセス権の付与に使用できます。
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: pod-reader
+rules:
+- apiGroups: [""] # "" はコアのAPIグループを示します
+  resources: ["pods"]
```

Comment:
Optionally, highlight that this is lowercase and plural. `Pod` is not equivalent, here, to `pods`.

Reply:
nah this is fine as-is